LibraBFT/Impl/Consensus/Types.agda

```diff
@@ -1,9 +1,11 @@
+{-# OPTIONS --allow-unsolved-metas #-}
```

Comment:
According to our [style guide](https://github.com/oracle/bft-consensus-agda/blob/main/STYLE.md#coding-style-conventions), the copyright notice should come before the pragma.

Reply:
FIXED